src/models/db/transfers.js

```diff
@@ -69,42 +71,64 @@ function convertToPersistent (data) {
     data.status_id = transferStatuses.getTransferStatusId(data.state)
     delete data.state
   }
-  data.transfer_id = data.id
+  data.transfer_uuid = data.id
   delete data.id
   return _.mapKeys(data, (value, key) => key.toUpperCase())
 }
 
-function * getTransfer (id, options) {
-  return db.selectOne({TRANSFER_ID: id}, options && options.transaction)
+function * getTransfer (uuid, options) {
+  return db.selectOne({TRANSFER_UUID: uuid}, options && options.transaction)
     .then((transfer) => {
-      return adjustments.getAdjustments(id, options).then((adjustments) => {
+      if (transfer === null) {
+        return null
+      }
+      return adjustments.getAdjustments(transfer._id, options)
+      .then((adjustments) => {
         const result = _.assign({}, transfer, adjustments)
-        return _.isEmpty(result) ? null : result
+        return _.isEmpty(result) ? null : _.omit(result, '_id')
```

Comment:
Then we wouldn't be able to use `selectOne`, which creates a lot more duplicated code